sw/device/tests/sim_dv/pwrmgr_deep_sleep_all_wake_ups.c

```diff
@@ -153,6 +159,30 @@ static void prgm_pinmux_wakeup(void *dif) {
                                                  detector_cfg));
 }
 
+/**
+ * usb config for test #4
+ * . Fake low power entry through usb
+ * . Force usb to output suspend indication
+ * (*dif) handle is not used but leave as is
+ * to be called from execute_test
+ */
+static void prgm_usb_wakeup(void *dif) {
+  usbdev_set_wake_module_active(true);
+  usbdev_force_dx_pullup(kDpSel, true);
+  usbdev_force_dx_pullup(kDnSel, false);
+
+  // give the hardware a chance to recognize the wakeup values are the same
+  busy_spin_micros(20);  // 20us
```

Comment:
From my past experience this approach is prone to failure in the nightly regression. I would suggest computing this delay using the `kClockFreqCpuHz` which is different for each platform.
I would look at the AON tests for a reference.